src/core/sync/mutex.d

```diff
@@ -128,13 +128,13 @@ class Mutex :
      * Throws:
      *  SyncError on error.
      */
-    @trusted void lock()
+    @trusted @nogc void lock()
```

Comment:
Please don't change that function, it's part of the public API, and inherited classes might not implement `@nogc`.